pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -441,7 +441,7 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 				Annotations: annotations,
 			}
 			causes := ValidateVirtualMachineInstanceMetadata(k8sfield.NewPath("metadata"), &vmi.ObjectMeta, config, "fake-account")
-			Expect(len(causes)).To(Equal(0))
+			Expect(causes).To(HaveLen(0))
```

Comment:
You can use `BeEmpty()` here.

Reply:
Done, thanks!

Reply:
@iholder-redhat Hi, I don't see the changes for the `BeEmpty()`, did you push them?

Reply:
Oops. You're right. thanks!
I've re-pushed